### PR TITLE
Game page map inline

### PR DESCRIPTION
**Pull request by yzemaze**
- map inline
- map-size calculation corrected to prevent pagebreaks due to map width set wider than necessary
- buy_companies wrapped & display: flex to avoid setting max-width (which could be a problem later on)
- min. 3 companies will be rendered inline (with rem <= 16px), max. depends on width of map
- route_selection slimmed

### CODE REVIEW

File: assets/app/view/game/company.rb [hunk 1]
```diff
@@ -65,14 +66,13 @@ def render
...
           style: {
-            display: 'inline-block',
             cursor: 'pointer',
             border: 'solid 1px gainsboro',
             'border-radius': '10px',
             overflow: 'hidden',
             padding: '0.5rem',
             margin: '0.5rem 0.5rem 0 0',
-            width: '300px',
+            width: '19rem',
```
Comment (tobymao) [inline assets/app/view/game/company.rb:75]: is this going to work on all browsers?

File: assets/app/view/game/map.rb [hunk 2]
```diff
@@ -92,9 +92,16 @@ def map_y
       end
 
       def render_map
-        w_size, h_size = @layout == :flat ? [85, 50] : [50, 85]
-        width = (@cols.size * w_size) + GAP
-        height = (@rows.size * h_size) + GAP
+        flat2flat = 85
+        edge_length = 50
```
Comment (tobymao) [inline assets/app/view/game/map.rb:96]: make these constants

i'm not sure what flat2flat means maybe choose a better name and use snake case

File: assets/app/view/game/company.rb (hunk 1 above)
Comment (yzemaze) [reply assets/app/view/game/company.rb:75]: I can’t test ’em all (FF, Chrome + Safari/iOS worked fine), but 19rem + 1rem padding = 20rem = width of corps and player cards. So if nobody did complain about them until now I wouldn’t expect any problems.
If font-size (user-setting in browser!) is set to 16px all these cards are 320px wide (+1 px border each side)

File: assets/app/view/game/map.rb (hunk 2 above)
Comment (yzemaze) [reply assets/app/view/game/map.rb:96]: flat2flat = flat side to flat side = [apothem](https://mathworld.wolfram.com/Apothem.html) * 2

**Pull request merged**